test_calcuator.py

```diff
@@ -0,0 +1,57 @@
+"""Calculator Tests."""
+
+import unittest
+import Calculator_exercise
```

Comment:
rather than import calculator exercise and call every function individually you should call the highest level function which does calculations and check that the ultimate answer is correct.

For the way you currently wrote the code it means you should call remove_spaces() each test and make sure you get the correct result for each operation. 

Given the necessary restructuring that you need to do to calculator file you should avoid making this change until you do the restructuring.